### DISCUSSION TITLE
第三方库同步编译

### DISCUSSION THREAD

**Discussion by plutolove** [xmake-io/xmake#5562] [Q&A]
依赖本地第三方库，需要调试第三方库，怎么可以改动第三方库之后可以一起同步编译，比如项目A依赖llvm，本地改了llvm，加一些debug信息，然后重新编译A，怎么可以同步编译llvm

**Comment by star-hengxing** [comment]
[policy](https://xmake.io/#/guide/build_policies?id=packageinstall_always) or `xmake require -h` 里面命令组合一下
